src/main/scala/firrtl/Namespace.scala

```diff
@@ -27,14 +27,15 @@ MODIFICATIONS.
 
 package firrtl
 
+import scala.collection.mutable
 import scala.collection.mutable.HashSet
```

Comment:
Could you also delete the import scala.collection.mutable.HashSet? I guess this can be a later style-fix commit deleting unnecessary imports